### PR TITLE
pyRiemann-qiskit v0.1.0

### PR DESCRIPTION
**Pull request by gcattan**
@sylvchev @qbarthelemy @toncho11 

I think we have enough material to deliver v0.1.0. We made good progress with the quantum MDM, enabled multiclass classification, and included more examples.

We have a few ideas for v0.1.1 that we can discuss : 
1) Neural network and quantum
2) Quantum emulation with copy classifier
3) Evaluation on real quantum backends (we have some credits to use thanks to the [IBM quantum research program](https://quantum-computing.ibm.com/credits-program))
4) Quantum art and visualization
5) Comparison of quantum vs classical computers for [bci illiteracy](https://github.com/NeuroTechX/moabb/blob/14d1ef2f7022a2e1923c0cf8d39efbab91adac34/moabb/datasets/compound_dataset/bi_illiteracy.py#L146)
6) ...

Let me know what you think.

### CODE REVIEW

**Review by qbarthelemy — COMMENTED** [review]
It is ok for me to release v0.1.0.

In the gallery, we could create a new item "Classification on other datasets", grouping the examples on Titanic dataset, multiclass EEG and dimension reduction. These three examples are neither ERP nor toy datasets classification examples.

@gcattan, keep the good practice of creating the branch in your personal fork, and not from the upstream repository.

**Comment by gcattan** [comment]
Yes, sorry about that, I will pay attention for the next PRs. 
Thank you for the review!

File: examples/other_datasets/README.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Classification on other datasets
+--------------------------------
+
+Contains examples with other datasets, including examples with non-EEG datasets (e.g. the titanic dataset).
```
Comment (qbarthelemy) [inline examples/other_datasets/README.txt:4]: In fact, `multiclass_classification.py` is an EEG-based example.
Do we split EEG examples according to the BCI paradigm (ERP, MI, etc)?

**Review by qbarthelemy — APPROVED** [review]

**Comment by gcattan** [comment]
@toncho11 I try to follow semantic versioning. In X.Y.Z, X is for major, Y for minor, and Z for patches.
X being 0, just means that the public API is not yet stable. 
But you are not wrong. Maybe it is stable enough to release 1.0.0 next time. 

Yes, I plan to release on github and pypi, but I would like to merge https://github.com/pyRiemann/pyRiemann-qiskit/pull/191 first.

**Pull request merged**